cli.js

```diff
@@ -88,6 +91,18 @@ function setDefaults(cli, configFileFlags) {
     compositeFlags.maximumFileSizeToCacheInBytes ||
     configFileFlags.maximumFileSizeToCacheInBytes;
 
+  // We can't just use ||, since compositeFlags.skipWaiting might be false.
+  compositeFlags.skipWaiting = ('skipWaiting' in compositeFlags) ?
+    compositeFlags.skipWaiting :
+    (('skipWaiting' in configFileFlags) ?
+      configFileFlags.skipWaiting : undefined);
+
+  // We can't just use ||, since compositeFlags.clientsClaim might be false.
+  compositeFlags.clientsClaim = ('clientsClaim' in compositeFlags) ?
+    compositeFlags.clientsClaim :
+    (('clientsClaim' in configFileFlags) ?
+      configFileFlags.clientsClaim : undefined);
+
```

Comment:
I personally find the nested ternary ops here a little harder to read, but let's keep as is. I think it should be fine.